db/system_distributed_keyspace.cc

```diff
@@ -407,10 +407,9 @@ system_distributed_keyspace::cdc_get_versioned_streams(context ctx) {
 
         for (auto& row : *cql_result) {
             auto ts = row.get_as<db_clock::time_point>("time");
-            auto exp = row.get_opt<db_clock::time_point>("expired");
```

Comment:
Should we remove "expired" from the schema as well? I might remember incorrectly but I think @elcallio had something to do with that field.

Reply:
Removing things from schemas requires extreme care.